src/lib/server.ts

```diff
@@ -106,6 +107,16 @@ export class Client {
       if (msg.command === DriverCommand.getLogConfig) {
         // We don't want to return transports since that's used internally.
         const { transports, ...partialLogConfig } = this.driver.getLogConfig();
+
+        if (
+          this.schemaVersion < 3 &&
+          typeof partialLogConfig.level === "string"
+        ) {
+          let levelNum = numberFromLogLevel(partialLogConfig.level);
+          if (levelNum != undefined) {
```

Comment:
Should we make it an error when it is undefined ?

Reply:
I guess it's the old schema, yolo.

Reply:
I suppose it wouldn't hurt as a best practices kinda thing

Reply:
oh that's right, this is coming from node-zwave-js, so I don't think we need to worry about it being undefined